Indicators/QuantConnect.Indicators.csproj

```diff
@@ -45,6 +45,8 @@
     <Reference Include="System.Xml" />
   </ItemGroup>
   <ItemGroup>
+    <Compile Include="StochasticOscillator.cs" />
```

Comment:
From Travis CI: CSC: error CS2001: Source file `StochasticOscillator.cs' could not be found

Please limit pull requests to a single change. I think @kaffeebrauer is handling the stochastics already.